app/templates/components/widgets/twitter-timeline.hbs

```diff
@@ -1,7 +1,50 @@
 {{#if this.handle}}
-  <a class="twitter-timeline" href="{{this.normalizedUrl}}" target="_blank" rel="noopener nofollow" data-height="500">
-    <i class="twitter icon"></i>
-    {{t 'Tweets by'}} @{{handle}}
-  </a>
-  <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
-{{/if}}
+  {{#unless this.device.isMobile}}
+    <a
+      class="twitter-timeline"
+      href="{{this.normalizedUrl}}"
+      target="_blank"
+      rel="noopener nofollow"
+      data-height="500"
+    >
+      <i class="twitter icon"></i>
+      {{t "Tweets by"}}
+      @
+      {{handle}}
+    </a>
+    <script
+      async
+      src="https://platform.twitter.com/widgets.js"
+      charset="utf-8"
+    ></script>
+  {{/unless}}
+  {{#if this.device.isMobile}}
+    <UiAccordion>
```

Comment:
there is a code duplication pls improve the code

Reply:
ke changes here instead make changes where the componenet is being used.

Reply:
> @ranjitkshah Pls make changes as asked above and I think test fail will also be fixed after that

changes done pls check